themes/default/content/blog/launch-redshift-into-aws-vpc-with-pulumi/index.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Launch an Amazon Redshift cluster into a VPC with Pulumi
+date: 2022-11-07
+meta_desc: In this post, you'll learn how to deploy an Amazon Redshift cluster into an AWS VPC and manage it with Pulumi.
+meta_image: meta.png
+tags:
+    - aws
+    - vpc
+    - redshift
+    - databases
+---
+
+Infrastructure as code (IaC) enables us to manage cloud resources using a declarative approach. We keep declarations of the cloud environment in the text file of the specific format. So, instead of provisioning cloud resources using the Amazon Web Console (AWS), we can set up our cloud environment using a command-line interface and a declaration. This allows us to keep declarations in the source control system and deploy cloud resources as we would with our application.
+
+AWS provides CloudFormation as an IaC tool. However, using CloudFormation isn’t always straightforward. We have a visual editor with the AWS console, enabling us to draw and configure cloud resources to deploy. This approach is very often difficult to learn and use.
+
+To solve this problem and accelerate IaC adoption, Pulumi offers the cross-platform IaC approach, which allows us to choose among the most popular scripting or programming languages, including TypeScript, Python, Go, C#, and Java. With Pulumi, we declare and configure our cloud resources as we would create programming objects in your application code. We can use IaC much faster without learning complicated tools like CloudFormation.
+
```

Comment:
I would just remove this entire section and provide more context on Redshift and datawarehouses. Might allude to the fact that Pulumi uses it as well.

Reply:
It seems to me there are two options. 

1/ say its from a 3rd party author and then keep a bunch of this other context
2/ make it all a bit more pulumified so removing references to cloudformation and adding a bit more context of the real-life use cases

Reply:
Yep, agreed. Option 2 would ultimately be better I think, but would take a bit of revising.